replay/session_handler.py

```diff
@@ -58,20 +58,20 @@ def get_spark_session(
     return spark
 
 
-def logger_with_settings() -> logging.Logger:
-    """Set up default logging"""
-    spark_logger = logging.getLogger("py4j")
-    spark_logger.setLevel(logging.WARN)
-    logger = logging.getLogger("replay")
-    formatter = logging.Formatter(
-        "%(asctime)s, %(name)s, %(levelname)s: %(message)s",
-        datefmt="%d-%b-%y %H:%M:%S",
-    )
-    hdlr = logging.StreamHandler()
-    hdlr.setFormatter(formatter)
-    logger.addHandler(hdlr)
-    logger.setLevel(logging.DEBUG)
-    return logger
+# def logger_with_settings() -> logging.Logger:
```

Comment:
Don't leave it like a comment, just remove it.
We have history to refer to if needed.